modules/virt-enabling-mac-address-pool-for-namespace-cli.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_vm_networking/virt-using-mac-address-pool-for-vms.adoc
+
+[id="virt-enabling-mac-address-pool-for-namespace-cli_{context}"]
+= Enabling a MAC address pool for a namespace in the CLI
+
+Enable a MAC address pool for virtual machines in a namespace by applying the `mutatevirtualmachines.kubemacpool.io=allocateForAll` label to the namespace.
+
+.Procedure
+
+* Add the KubeMacPool label to the namespace.
+The following example adds the KubeMacPool label to two namespaces, `<namespace1>` and `<namespace2>`:
++
+----
+$ oc label namespace <namespace1> <namespace2> mutatevirtualmachines.kubemacpool.io=allocateForAll
```

Comment:
I think we should also add an important **note** that is also on the [u/s documentation ](https://github.com/k8snetworkplumbingwg/kubemacpool#how-to-enabledisable-kubemacpool-for-a-namespace):
`note: If a VMI is created directly and not through a VM, then it will not be handled by kubemacpool.`
(if already added then sorry for re-asking)

Reply:
Thanks for pointing that out @RamLavi, I had not already added it (but just have).
I actually thought that note in the upstream meant that an (unattached) VMI still gets handled by KubeMacPool but that there's an extra step (the Pod Handler) to get there. Do you think it's worthwhile updating the upstream with what you suggested here? I'm happy to create the PR on upstream docs if so.

Reply:
in u/s KMP the pod handler is operational.
It is only because in d/s - the pod handler is disabled - then the vmi's without a vm don't get allocated.
so we need to make it clear in the d/s/ documentation...

Reply:
Ah I see. Thanks for the clarification.